xla/stream_executor/rocm/hip_blas_lt.cc

```diff
@@ -130,6 +130,12 @@ static absl::StatusOr<hipblasLtEpilogue_t> AsHipblasLtEpilogue(
       return HIPBLASLT_EPILOGUE_GELU_BIAS;
     case gpu::BlasLt::Epilogue::kBiasThenGELUWithAux:
       return HIPBLASLT_EPILOGUE_GELU_AUX_BIAS;
+#endif
+#if TF_ROCM_VERSION >= 60400
```

Comment:
@alekstheod  this is problematic as we don't have the official ROCm6.4 release yet, it should  `> 60400` or `>= 60300` to protect  @Ruturaj4 could you merge this https://github.com/ROCm/xla/commit/4e717251dbacc202a43547b5b9e9abf40612999c to 0.5.0?